cmu_graphics/cmu_graphics.py

```diff
@@ -1222,8 +1224,52 @@ def __setattr__(self, attr, value):
             return self._app.__setattr__(attr, value)
         return super().__setattr__(attr, value)
 
+def processRunAppArgs(args, kwargs):
```

Comment:
The body of this function seems to be oddly indented with 8 spaces. It's a nitpick, but can you tab out once here?